pkg/object/scs.go

```diff
@@ -89,6 +90,9 @@ func (s *scsClient) List(prefix, marker string, limit int64) ([]*Object, error)
 	}
 	s.marker = list.NextMarker
 	n := len(list.Contents)
+	// 咨询scs技术支持,每页内的key“可能”顺序有误
```

Comment:
Can you translate the comment into English?

Reply:
Sorry. I was careless. Please review again.